README.md

```diff
@@ -1,3 +1,17 @@
+# english
+**SendScriptWhatsApp**
```

Comment:
It would be better, to have the title as "SendScriptWhatsApp/EN" and in the portuguese version the same thing with "/PT" at the end.

Reply:
okay! so now is it alright? and can it be now merged. I am learning open source right now . Keep coding.